lib/cbus_elixir_web/router.ex

```diff
@@ -19,8 +19,8 @@ defmodule CbusElixirWeb.Router do
     get "/", PageController, :index
   end
 
-  # Other scopes may use custom stacks.
-  # scope "/api", CbusElixirWeb do
-  #   pipe_through :api
-  # end
+  scope "/api", CbusElixirWeb do
+    pipe_through :api
+    resources "/meetings", MeetingController, except: [:new, :edit]
```

Comment:
I don't think we need routes; we'll just generate the meeting dates and we're done.

Reply:
That was all from the generator. I don't need any resources at all?

Reply:
Nope, no routes. We won't be managing Meetings at all outside of the initial generation